docs/conf.py

```diff
@@ -22,114 +16,52 @@
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-
-# The short X.Y version.
-# Note: We use the full version in both cases.
-version = distro.__version__
-
-# The full version, including alpha/beta/rc tags.
-release = version
-
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = None
-
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ["tests", ".tox", ".git", "build_docs", "ld.egg-info"]
-
-# If true, '()' will be appended to :func: etc. cross-reference text.
-add_function_parentheses = True
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
-
-# If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = True
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
 
-# -- Options for HTML output ----------------------------------------------
+# -- Options for HTML output -------------------------------------------------
 
-# The theme to use for HTML and HTML Help pages.
-# See http://www.sphinx-doc.org/en/stable/theming.html for built-in themes.
+# The theme to use for HTML and HTML Help pages. See
+# https://www.sphinx-doc.org/en/master/usage/theming.html or a list of builtin
+# themes.
 html_theme = "classic"
 
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.
-# See http://www.sphinx-doc.org/en/stable/theming.html for the options
-# available for built-in themes.
-html_theme_options = {}
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ["html_static"]
-
-# Add any extra paths that contain custom files (such as robots.txt or
-# .htaccess) here, relative to this directory. These files are copied
-# directly to the root of the documentation.
-html_extra_path = ["html_extra"]
-
-# Output file base name for HTML help builder.
-htmlhelp_basename = "distro_doc"
 
 # -- Options for intersphinx extension ------------------------------------
 # For documentation, see
-# http://www.sphinx-doc.org/en/stable/ext/intersphinx.html
+# https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
 
 # Defines the prefixes for intersphinx links and the targets they resolve to.
 # Use Python 3.7 as that is the last version to include
 # platform.linux_distribution() and platform.dist(). Example RST source for
 # 'py' prefix:
 #     :py:func:`platform.dist`
 intersphinx_mapping = {"py": ("https://docs.python.org/3.7", None)}
```

Comment:
```suggestion
intersphinx_mapping = {"py": ("https://docs.python.org/3", None)}
```

Reply:
iirc this should also work and is better suited for the 'future'. i know you didn't touch this, but it fits the purpose imo. the comment above would also need to be adjusted.

Reply:
See f7371a42f442e8223b4c0a3318c928a2845030fd

This was intentionally switched to Python 3.7 as newer docs don't have `platform.linux_distrobution` anymore. The comment a few lines up also mentions this. Let me know if it can be expanded to be more clear.